build/deploy.js

```diff
@@ -24,18 +24,6 @@ const deployToFirebase = (cb) => {
     return
   }
 
-  debug('Installing firebase-tools...')
```

Comment:
Lets actually delete `deploy.js` and the npm script that calls it since it won't be used